students/seyong/users/views.py

```diff
@@ -1,3 +1,44 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
+from django.http  import JsonResponse
+
+from django.views import View
+from users.models  import User
```

Comment:
이 부분도 항상 신경 써주세요 세용님!
```suggestion
from django.http  import JsonResponse
from django.views import View

from users.models  import User
```